src/Components/Aspire.Microsoft.Azure.Cosmos/AspireMicrosoftAzureCosmosDBExtensions.cs

```diff
@@ -101,6 +101,13 @@ public static class AspireMicrosoftAzureCosmosDBExtensions
 
         configureClientOptions?.Invoke(clientOptions);
 
+        var cosmosApplicationName = CosmosConstants.CosmosApplicationName;
+        if (!string.IsNullOrEmpty(clientOptions.ApplicationName))
+        {
+            cosmosApplicationName += clientOptions.ApplicationName;
```

Comment:
nit: Add a separator

Reply:
Added the `|` separator at the moment.

Reply:
Is the `|` the correct separator?

https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy

Seems to suggest using a slash might be preferred?